src/aliceVision/hdr/brackets.cpp

```diff
@@ -154,7 +154,8 @@ void selectTargetViews(std::vector<std::shared_ptr<sfmData::View>> & out_targetV
         {
             lines.push_back(line);
         }
-        if ((lines.size() < 3) || (std::stoi(lines[0].c_str()) != groups.size()) || (std::stoi(lines[1].c_str()) < groups[0].size()))
+        if ((lines.size() < 3) || (std::stoi(lines[0].c_str()) != groups.size()) || (std::stoi(lines[1].c_str()) < groups[0].size()) ||
```

Comment:
stoi takes a string in input, so no need for c_str.
At some point, I fixed it in the whole project but it seems that it is back again.

Reply:
Done